doc/source/examples/incompressible-flow/3d_nitsche_mixer_with_pbt_impeller/nitscher_mixer_with_pbt_impeller.rst

```diff
@@ -0,0 +1,279 @@
+	  subsection mesh
+      		set type                 = gmsh
+      		set file name            = pbt.msh
+          set simplex = true
+	  end
+    subsection solid velocity
+	    set Function expression = 0 ; 1*2*3.1416*z ; -1*2*3.1416*y
+    end
+
+   	set beta 			= 10
+    set enable particles motion		= true
+    set particles sub iterations      = 10
+    set calculate torque on solid = true
+  end
+
+First we specify the number of immersed solid objects. In our case, since we only have the PBT, we set ``number of solids=1``.  Setting ``verbosity = true`` enables us to output in the terminal the post-processing done on the solid object.
+
+For each Nitsche solid, multiple parameters have to be provided. First, the ``mesh`` subsection works just like a regular ``mesh`` subsection, but instead controls the mesh used for the Nitsche IB. In the present case, we import a GMSH mesh of the PBT made using simplices (tetrahedra). Hence, we set ``simplex = true``.
+
+.. note::
+  
+  The Nitsche IB method works by forcing the fluid at the location of the gauss points of the solid triangulation in order to apply the no-slip boundary condition within the solid object. Consequently, the solid mesh should have a characteristic size which is of the same order as that used for the fluid dynamics problem. Using a finer mesh will not cause any problems, but will increase the computational cost without adding any benefits.
+
+
+We specify the ``solid velocity`` using a user defined function. In the present case, the velocity that we specify induces a rotation around the :math:`x` axis of the meshes, centered at the point :math:`(0,0,0)`. This corresponds to the location of the center of our PBT in the accompanying GMSH mesh.
+
+The ``beta`` parameter controls the intensity of the application of Nitsche method in the fluid region. Setting a higher value of ``beta`` leads to a stiffer problem, but prevents the fluid from penetrating the solid. For flow at Reynolds number larger than one, we have found that setting ``beta`` between 10 and 1000 leads to satisfactory results. When the flow is more viscous, higher values of ``beta`` should be used to compensate for the larger shear stresses acting on the immersed solid.
+
+Nitsche method can be used with both static and dynamic immersed solid. When the velocity imposed on the solid leads to a motion of the solid, we must set ``enable particles motion=true`` . This option should only be set to true on the solids which are actually moving, since it adds a non-negligible cost to the simulations. To ensure that particles are always located efficiently as they move through cell, we set ``particles sub iterations = 10``. This enables us to displace the particle using 10 sub-steps, thus facilitating the detection of the cells in which they lie. Generally, it is a good practice to have sufficient particles sub iterations so as to ensure that particles do not move more than half a cell during a particle sub iteration.
```

Comment:
```suggestion
Nitsche method can be used with both static and dynamic immersed solid. When the velocity imposed on the solid leads to a motion of the solid, we must set ``enable particles motion=true`` . This option should only be set to true on the moving solids since it adds a non-negligible cost to the simulations. To ensure that particles are always located efficiently as they move through the cell, we set ``particles sub iterations = 10``. This enables particle displacing using 10 sub-steps, thus facilitating the detection of the cells in which they lie. Generally, it is a good practice to have sufficient particles sub iterations so as to ensure that particles do not move more than half a cell during a particle sub iteration.
```